wiki/views/article.py

```diff
@@ -89,7 +89,7 @@ def form_valid(self, form):
                                     'other_read': self.article.other_read,
                                     'other_write': self.article.other_write,
                                     })
-                messages.success(self.request, _(u"New article '%s' created.") % self.newpath.article.current_revision.title)
+                messages.success(self.request, _(u"New article %s created.") % self.newpath.article.current_revision.title)
```

Comment:
@madnihamza1841 could you please replace single quote with their htmlentities instead of removing them?
```suggestion
                messages.success(self.request, _(u"New article &quot;%s&quot; created.") % self.newpath.article.current_revision.title)
```

Reply:
@ziafazal I have already tried and tested this case. Does not seem to work. Also tried and tested &apos, &#39;, \' and \" 

![image](https://user-images.githubusercontent.com/86109301/136029447-6a2328f4-e335-4d6c-8d22-6514f01494f8.png)

Reply:
@madnihamza1841 I think try another approach which is wrapping this [div](https://github.com/edly-io/django-wiki/blob/edx_release/wiki/templates/wiki/base.html#L160) with `autoescape` tage
 ```suggestion
                {% autoescape off %}
    <div class="alert alert-{{ message.tags }}">
            <a class="close" data-dismiss="alert" href="#">&times;</a>
            {{ message }}
          </div>
{% endautoescape %}
```